pkg/collector/corechecks/sbom/processor.go

```diff
@@ -189,36 +189,47 @@ func (p *processor) processHostRefresh() {
 
 	go func() {
 		result := <-ch
+		log.Debugf("Sending sbom to event platform: %v", result)
```

Comment:
Not a big deal but the comment is a bit misleading as it's not directly sent to the backend

Reply:
Nice catch ! thx